### PR TITLE
test/mpi: Fix argument passing to pt2pt_large test

### PR DESCRIPTION
**Pull request by raffenet**
## Pull Request Description

Need to prefix arguments meant for test binaries with arg=.
[2024-09-17] added commit to fix issue exposed by corrected test.

## Author Checklist
* [x] **Provide Description** 
      Particularly focus on _why_, not _what_. Reference background, issues, test failures, xfail entries, etc.
* [x] **Commits Follow Good Practice**
      Commits are self-contained and do not do two things at once. 
      Commit message is of the form: `module: short description` 
      Commit message explains what's in the commit.
* [ ] **Passes All Tests**
      Whitespace checker. Warnings test. Additional tests via comments.
* [x] **Contribution Agreement**
      For non-Argonne authors, check [contribution agreement](http://www.mpich.org/documentation/contributor-docs/). 
      If necessary, request an explicit comment from your companies PR approval manager.

### CODE REVIEW

**Comment by raffenet** [comment]
test:mpich/ch4/most
test:mpich/ch3/most

**Review by hzhou — APPROVED** [review]
LGTM

**Comment by raffenet** [comment]
test:mpich/ch4/most
test:mpich/ch3/most

**Comment by raffenet** [comment]
test:mpich/ch4/most
test:mpich/ch3/most

File: src/mpid/ch3/src/ch3u_request.c [hunk 1]
```diff
@@ -393,15 +393,15 @@ int MPIDI_CH3U_Request_load_recv_iov(MPIR_Request * const rreq)
...
 {
     MPI_Aint last;
-    int tmpbuf_last;
+    MPI_Aint tmpbuf_last;
     int mpi_errno = MPI_SUCCESS;
     
     MPIR_FUNC_ENTER;
 
-    tmpbuf_last = (int)(rreq->dev.msg_offset + rreq->dev.tmpbuf_sz);
+    tmpbuf_last = rreq->dev.msg_offset + rreq->dev.tmpbuf_sz;
```
Comment (hzhou) [inline src/mpid/ch3/src/ch3u_request.c:401]: Looks good! I wonder why our tests do not consistently catch this.

**Review by hzhou — APPROVED** [review]
LGTM.

**Pull request merged**